src/doc/trpl/closures.md

```diff
@@ -33,8 +33,8 @@ let plus_two = |x| {
 assert_eq!(4, plus_two(2));
 ```
 
-You’ll notice a few things about closures that are a bit different than regular
-functions defined with `fn`. The first of which is that we did not need to
+You’ll notice a few things about closures that are a bit different to regular
```

Comment:
'to' reads very strangely to me. 'from' would sound good though. What do you think?